client/src/containers/SignIn/SignIn.js

```diff
@@ -1,24 +1,53 @@
-import React from 'react';
+import React, { useState, useEffect } from 'react';
 import { Link } from 'react-router-dom'
 import SmallContainer from '../../components/SmallContainer/SmallContainer';
 import InputField from '../../components/InputField/InputField';
+import { getError, hasError, isTouched, handleErrors } from '../../libs/validate';
+import { schema, inputs } from './Schema';
+import { handleChange, handleBlur } from '../../libs/handleInputEvents';
+import logo from '../../constants/constants';
+import { handleSubmit } from './utils';
 import './SignIn.css';
 
-function SignIn() {
+const SignIn = (props) => {
+  const [ user, setUser ] = useState({ name: '', email: '', password: '' });
```

Comment:
You can create a custom hook and use that both forms, it will be better than duplicating the code.